src/zcash/GenerateParams.cpp

```diff
@@ -0,0 +1,21 @@
+#include "zcash/JoinSplit.hpp"
+
+#include <iostream>
+
+int main(int argc, char **argv)
+{
```

Comment:
Maybe print a warning that this is not secure for generating production parameters. (There's no way to prove that you didn't keep a backdoor.)